core/bones/string.py

```diff
@@ -42,11 +50,20 @@ def isEmpty(self, value):
 
         return not bool(str(value).strip())
 
+    def isInvalid(self, value):
+        """
+        Returns None if the value would be valid for
+        this bone, an error-message otherwise.
+        """
+        if self.maxLength is not None and len(value) > self.maxLength:
+            return "Maximum length exceeded"
+        return None
+
     def singleValueFromClient(self, value, skel, name, origData):
         value = utils.escapeString(value, self.maxLength)
         err = self.isInvalid(value)
         if not err:
```

Comment:
```suggestion
        if not (err := self.isInvalid(value)):
```
We can shortcut this too.